aiohttp/test_utils.py

```diff
@@ -673,10 +673,10 @@ def make_mocked_request(
 
     if writer is None:
         writer = mock.Mock()
```

Comment:
Ideally, this would be replace with mock.create_autospec().
If that's too much to figure out, then I think AsyncMock should atleast be using `return_value=None`. I don't think you should be passing any positional arguments to it.